typings/@langchain/langgraph/index.d.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+declare module '@langchain/langgraph';
```

Comment:
Asking to make sure: Did we check, and are we sure there's not typings npm modules already available for this? the `typings` folder is a remain of an ancien era, in theory in 2024 we should need to add anything to it.

Reply:
Thank you @pgayvallet, types exists, but are not compatible with TS@4 😢 
https://github.com/langchain-ai/langgraphjs/pull/213

Reply:
:sadpanda: